docs/web-apps/automated-testing/selenium/selenium4.md

```diff
@@ -47,41 +50,40 @@ Below are features new to Selenium 4 that may impact your tests.
 </tr>
 <tr>
 <td>New Element Attribute and Property methods</td>
-<td>Two new methods have been added to more precisely get a given element attribute or a property. These are especially useful for performance and preciseness in Sauce Labs tests. <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/AttributePropertyTest.java">Code example</a>.</td>
+<td>Two new methods have been added to more precisely get a given element attribute or a property. These are especially useful for performance and preciseness in Sauce Labs tests. <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/AttributePropertyTest.java">See a code example</a>.</td>
 </tr>
 </table>
 
 
 ## Preparing Your Test Code
 
-As mentioned previously, Selenium 4 deprecates support for legacy JSON wire protocol and its static browser methods `DesiredCapabilities`. It will be replaced by W3C WebDriver-compliant protocol and the use of `browserOptions`. Using `browserOptions` simplifies the configuration needed to start a new session, allows you to set browser-specific settings (like headless in Chrome), and reduces the chances of browser misconfiguration.
+Because Selenium 4 deprecates support for legacy JWP (`DesiredCapabilities`) &#8212; replacing it with W3C WebDriver-compliant (`browserOptions`) &#8212; you'll need to prepare your test code accordingly. Using W3C `browserOptions`:
+* Simplifies the configuration needed to start a new session.
+* Allows you to set browser-specific settings (like headless in Chrome).
+* Reduces the chances of browser misconfiguration.
 
-### W3C Capabilities
+### W3C WebDriver-Compliant Capabilities
 
-In Selenium 4, W3C capabilities will be supported by default under the hood. Your test capabilities must be structured to be W3C-compliant, otherwise they'll be ignored.
-
-Here is the list of [W3C WebDriver standard capabilities](https://www.w3.org/TR/webdriver1/#capabilities). Any capability not in this list must include a vendor prefix. This rule applies to browser-specific capabilities as well as Sauce Labs-specific capabilities. For example, if you want use the `build` and `name` capabilities in for your Sauce Labs tests, you'll need to wrap them in a `sauce:options` block (a complete example is shown below). For more information and examples, see our [W3C WebDriver Capabilities Support](/dev/w3c-webdriver-capabilities) doc.
-
-Below are examples showing the switch from `DesiredCapabilities` to `browserOptions`.
+You must structure your test capabilities to be W3C-compliant and switch from `DesiredCapabilities` (JWP) to `browserOptions` (W3C), otherwise they'll be ignored.
```

Comment:
These tabs for the different browser options are brilliant, I love it!

Reply:
thank you!